app/Http/Controllers/NewsController.php

```diff
@@ -52,6 +116,53 @@ public function index()
         }
     }
 
+    /**
+     * Get News Post
+     *
+     * Returns details of the specified news post.
+     *
+     * ---
+     *
+     * ### Response Format
+     *
+     * Returns a [NewsPost](#newspost) with `content` and `navigation` included. Querying by ID will return a redirect to query by slug.
```

Comment:
The redirect should probably be skipped for api request instead to match user api behavior